frontend/src/app/components/fhir-card/resources/observation/observation.stories.ts

```diff
@@ -40,30 +34,11 @@ const meta: Meta<ObservationComponent> = {
 export default meta;
 type Story = StoryObj<ObservationComponent>;
 
-// More on writing stories with args: https://storybook.js.org/docs/angular/writing-stories/args
-let r4Example1DisplayModel =  new ObservationModel(R4Example1Json, fhirVersions.R4)
-r4Example1DisplayModel.source_id = '123-456-789'
-r4Example1DisplayModel.source_resource_id = '123-456-789'
-export const R4Example1: Story = {
-  args: {
-    displayModel: r4Example1DisplayModel
-  }
-};
-
-let r4Example2DisplayModel =  new ObservationModel(R4Example2Json, fhirVersions.R4)
-r4Example2DisplayModel.source_id = '123-456-789'
-r4Example2DisplayModel.source_resource_id = '123-456-789'
-export const R4Example2: Story = {
-  args: {
-    displayModel: r4Example2DisplayModel
-  }
-};
-
-let r4Example3DisplayModel =  new ObservationModel(R4Example3Json, fhirVersions.R4)
-r4Example3DisplayModel.source_id = '123-456-789'
-r4Example3DisplayModel.source_resource_id = '123-456-789'
-export const R4Example3: Story = {
+let observation = new ObservationModel(observationR4Factory.referenceRange().build(), fhirVersions.R4);
+observation.source_id = '123-456-789'
```

Comment:
If I'm reading this right, the 3 example fixtures have been replaced with the Observation Factory. 

But in the Factory I only see 1 "Observation" defined. 
These examples contain a bunch of variations for the Observation model. Ideally I'd like to ensure they are used. Can we have the Observation Factory read from files -- rather than hardcode in a single Observation?

Reply:
> If I'm reading this right, the 3 example fixtures have been replaced with the Observation Factory.

Only kind of. I removed the other stories from this file because I added more examples to the new observation-bar-chart-component stories. The first two example JSONs both used valueQuantity and only the second one had a range. So I basically switched to use the factory with valueQuantity and range so that it would verify that it displays the chart (which is more thoroughly tested in its own stories file) and displays both the value and reference in the table like it was before. 

Admittedly I don't have a replacement for that third file yet which was using codeable concept, but I was planning on adding better parsing for that in upcoming PRs. But I could add that example back for now if you'd prefer. I didn't think it would be much of a miss until code was updated since the display doesn't actually work for that observation yet. It isn't something that can be displayed as a bar chart (`"text": "YES"`) and no values are displayed in the table.

> These examples contain a bunch of variations for the Observation model. Ideally I'd like to ensure they are used. Can we have the Observation Factory read from files -- rather than hardcode in a single Observation?

I could probably figure out a way to import from a file, but getting away from the JSON examples is part of the reason I added Fishery. The way example files are named right now, there is no way to know what exactly the example is for without manually looking into it and parsing the entire file to figure out why one is different from the other and which one should be used.

With Fishery, there may only be one base the way it is written right now, but you can define traits that edit the base to get you exactly what you want in a way that I would argue is easier to use, reuse, and read. 

For example, when testing how reference ranges look in a chart, instead of adding 6 new files, I'm able to do:
```
// all the different reference range variations
observationR4Factory.referenceRange().build()
observationR4Factory.referenceRangeOnlyLow().build()
observationR4Factory.referenceRangeStringOnlyLow().build()
observationR4Factory.referenceRangeOnlyHigh().build()
observationR4Factory.referenceRangeStringOnlyHigh().build()

// using both a specific value variation and a specific reference range variation 
// with custom numbers to test an abnormal lab result
observationR4Factory.valueString('9.9 units').referenceRangeStringOnlyHigh(8).build()
```

I think adding variations via traits rather than new files is going to be more readable and reusable. It might require a little more work to pull out the needed code and put it into a trait, but I think it will be worth it for testing.

Also, just to note, in response to:
> its hard to update, most test observations we'll find in the wild are JSON blobs, which are easy to copy paste into a new file, but converting to a TS Object is a bit annoying.

We don't actually need to convert to TS. I did for the example that I created purely because I thought it looked nicer than JSON and with keyboard shortcuts it wasn't much of an issue for me. But copy pasting JSON in as the base and/or traits works just fine. TS is treating it all as a hash either way.

Lol, if I haven't convinced you of factories yet, let me know and I'll look into loading from different JSON files as the base.

Reply:
nah, I'm sold. I was just worried that it would make copy-pasting from HL7 examples & testing docs harder. Sounds like it'll be worth it though.